swift/Tests/Blockchains/SolanaTests.swift

```diff
@@ -128,4 +128,55 @@ class SolanaTests: XCTestCase {
         XCTAssertEqual(output.encoded, expectedString)
     }
 
+    func testCreateTokenAccountSigner() throws {
+        let createAccountMessage = SolanaCreateTokenAccount.with {
+            $0.tokenMintAddress = "SRMuApVNdxXokk5GT7XD5cUUgXMBCoAz2LHeuAoKWRt"
+            $0.tokenAddress = "EDNd1ycsydWYwVmrYZvqYazFqwk1QjBgAUKFjBoz1jKP"
```

Comment:
can we calculate this address from private key?

Reply:
Yes, but exactly how, is still not finalized.  Either method in wallet core, or through RPC.  But better keep this outside.